infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationClientRouter.java

```diff
@@ -136,7 +134,7 @@ public LogReplicationClientRouter(LogReplicationRuntimeParameters parameters,
         this.remoteClusterDescriptor = parameters.getRemoteClusterDescriptor();
         this.remoteClusterId = remoteClusterDescriptor.getClusterId();
         this.parameters = parameters;
-        this.timeoutResponse = parameters.getRequestTimeout().toMillis();
+        TIMEOUT_RESPONSE = parameters.getRequestTimeout().toMillis();
```

Comment:
I think we need to keep the original timeout. Instead if you want to set larger timeouts for specific requests you can overload ` sendRequestAndGetCompletable`  to pass a timeout value